MonoGame.Framework.Content.Pipeline/Graphics/PixelBitmapContent.cs

```diff
@@ -17,7 +17,7 @@ public class PixelBitmapContent<T> : BitmapContent where T : struct, IEquatable<
 
         internal SurfaceFormat _format;
 
-        public PixelBitmapContent(int width, int height)
+        public PixelBitmapContent(int width, int height, IContentContext context) : base(context)
```

Comment:
Adding these additional arguments will break the public API compatibility with XNA (see https://learn.microsoft.com/en-us/previous-versions/windows/xna/bb195980(v=xnagamestudio.10)). 

We might need a different way to get the "current" context, perhaps a static factory class or something similar?

Reply:
Oh man, I completely spaced and forgot about api compatibility. _Whoops_. Thanks for check! I'll set about coming up with alternatives. I like the suggestions you gave.